src/ontology/uberon.Makefile

```diff
@@ -1170,7 +1170,7 @@ $(TMPDIR)/uberon-taxmod-%.owl: ext.owl
 # ----------------------------------------
 
 # Generate cross-references from the FBbt mapping file
-$(COMPONENTSDIR)/mappings.owl: mappings/fbbt-mappings.sssom.tsv ../scripts/sssom2xrefs.awk
+$(OTHER_SRC): mappings/fbbt-mappings.sssom.tsv ../scripts/sssom2xrefs.awk
```

Comment:
This is not future-proof. This works for now while `mappings.owl` is the only component, but won‘t work anymore if/when we add another component in the future.

I think it’s fine here to leave that rule as it were; that is, keep referring to the mappings component directly by its filename, not through the `OTHER_SRC` variable.